src/components/form/HdInputPhone.vue

```diff
@@ -0,0 +1,368 @@
+<template>
+  <div class="phone-input">
+    <button
+      class="phone-input__selector"
+      aria-haspopup="listbox"
+      aria-label="Choose a country code:"
+      @click="toggleDropdown"
+      :aria-activedescendant="`${selectedCountry.dial_code}, ${t.COUNTRIES[selectedCountry.code]}`"
+    >
+      <span :class="`phone-input__selector__flag flag-icon flag-icon-${selectedCountry.code.toLowerCase()}`"></span>
+      <HdIcon
+        width="24px"
+        height="24px"
+        :src="smallArrowIcon"
+        :class="arrowClassNames"
+      />
+    </button>
+    <ul
+      id="country-codes-list"
```

Comment:
Do we need this `id` attribute?

Reply:
Actually I put it there while I was adding the aria labels, but I was changing the structure while I was coding, so in the end I think that I forgot why it was there at the beginning 😅 I think I will just remove it